lib/ot.js

```diff
@@ -197,3 +200,28 @@ exports.transformPresence = function(presence, op, isOwnOp) {
 
   presence.v++;
 };
+
+/**
+ * json0 had a breaking change in https://github.com/ottypes/json0/pull/40
+ * The change added stricter type checking, which breaks fetchSnapshot()
+ * when trying to rebuild a snapshot from old, committed ops that didn't
+ * have this stricter validation. This method fixes up the op paths to
+ * pass the stricter validation
+ */
+function normalizeJson0Paths(snapshot, json0op) {
+  if (snapshot.type !== types.defaultType.uri) return;
+  var ops = json0op.op;
```

Comment:
Change to either `opComponents` or `components` to distinguish between this and "top-level" ops